src/website/app/demos/Tabs/examples/Tab/maxWidth.js

```diff
@@ -0,0 +1,28 @@
+/* @flow */
+import Tabs, { Tab } from '../../../../../../library/Tabs';
+import Text from '../../../../../../library/Text';
+import content from '../../shared/content';
+
+export default {
+  id: 'max-width',
+  title: 'Maximum Width',
+  description: `Use the \`maxWidth\` prop to constrain the width of the Tab.
+Note that if the Tab \`title\` is longer than the maximum width, it will be
+truncated but accessible via [Tooltip](/components/tooltip). Also note that if
```

Comment:
I am unable to activate these tooltips on mobile (iPhone).  I can also reproduce using Chrome dev tools device toolbar.

Reply:
We believe this is a limitation of Tooltip more so than Tabs. If it is to be improved, it should be a separate effort.